backend/src/ml_space_lambda/app_configuration/lambda_functions.py

```diff
@@ -0,0 +1,62 @@
+#
+#   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License").
+#   You may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import json
+import logging
+
+from ml_space_lambda.data_access_objects.app_configuration import AppConfigurationDAO, AppConfigurationModel, SettingsModel
+from ml_space_lambda.utils.common_functions import api_wrapper
+
+log = logging.getLogger(__name__)
+app_configuration_dao = AppConfigurationDAO()
+
+
+@api_wrapper
+def update_configuration(event, context):
```

Comment:
Where do users call this api from? Are we going to have some kind of admin page in MLSpace where this can be edited?

Reply:
An upcoming change will be to update/improve our existing admin page such that there's a section for in-app configurations. This will be where this update API is invoked from. It's also where we invoke the get API and request the 5 latest versions so admins can look at prior versions.

In the distant future we'll also create a ProjectOwner page where project owners can see some of these fields and have a project-specific configuration, separate from the app-wide one.

Reply:
will this admin page be accessible before setting up OIDC, removing the need for the command line setup wizard?

Reply:
OIDC is necessary to log into the app and is part of configuration at deployment time. The in-app configuration changes will deal with configurations that only affect the app after deployment (such as which services are enabled or disabled). The config setup wizard is still necessary since it deals exclusively with deployment-time configuration.